src/packer.json

```diff
@@ -29,13 +29,13 @@
       "region_kms_key_ids": {},
       "source_ami_filter": {
         "filters": {
-          "name": "debian-10-amd64-*",
+          "name": "debian-11-amd64-*",
           "root-device-type": "ebs",
           "virtualization-type": "hvm"
         },
         "most_recent": true,
         "owners": [
-          "136693071363"
+          "903794441882"
```

Comment:
Eventually an official Debian 11 AMI will be released, at which point you will probably need to revert this owner account change.  I am now checking monthly that our base AMIs will still match the most recent (there is a recurring calendar item) so this should eventually get picked up by that.

Reply:
Good call - thanks for the reminder.  When I have a few minutes, I will create issues about that for this repo and for [cisagov/pca-gophish-composition-packer](https://github.com/cisagov/pca-gophish-composition-packer), which is in a similar situation.

Reply:
Issues documented in https://github.com/cisagov/assessor-portal-packer/issues/2 and https://github.com/cisagov/pca-gophish-composition-packer/issues/56.